packages/babel-generator/src/buffer.js

```diff
@@ -54,7 +54,7 @@ export default class Buffer {
         configurable: true,
         enumerable: true,
         get() {
-          return this.map = map.get();
+          return (this.map = map.get());
```

Comment:
We add parenthesis around all the assignment in order to call attention to the fact that this expression is mutating a variable.